base/src/main/java/com/smartdevicelink/managers/screen/choiceset/KeyboardListener.java

```diff
@@ -84,4 +84,6 @@
      * @param currentInputText - The user's full current input text
      */
     void onKeyboardDidSendEvent(KeyboardEvent event, String currentInputText);
+
+    void onMaskHasChanged(KeyboardEvent event);
```

Comment:
@kostyaBoss On line 88 I would rename this to `onKeyboardInputMaskHasChanged()`.